app/controllers/api/v1/portfolio_items_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
           collection(Tag.find(params.require(:tag_id)).portfolio_items)
         else
           authorize(PortfolioItem)
-          collection(policy_scope(PortfolioItem.all))
+          collection(policy_scope(PortfolioItem.all), true)
```

Comment:
@eclarizio Wouldn't this end up displaying all the PortfolioItems (Products) as opposed to limiting it by the read access to Portfolios?

Reply:
@mkanoor It shouldn't, the `policy_scope` calls https://github.com/RedHatInsights/catalog-api/blob/master/app/policies/portfolio_item_policy.rb#L30 which should ensure that it only returns the ones that the user is allowed to see based on portfolio ids that they are able to read.